docs/source/technical_tutorials/search/maap_stac_r.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    http_urls = sapply(stac_query$features, function(x) {x$links[[5]]$href})
</pre><p>I would either remove the <code>http_urls</code> or section or make a note that these links are for the STAC item metadata (links for <code>rel='self'</code> I think).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Oops, totally didn't realize those were the metadata files - I removed them!</p>